### ISSUE TITLE
Task run time display on tentacle upgrade screen isn't accurate

### ISSUE THREAD

**Issue by NickJosevski** [OctopusDeploy/Issues#3422]
See screenshot:
for `ui-4-WIN-1KKE3DLFQP7-1`

logs show:
```
started at 11:35.47
finished at 11:36:23
```
which is `36 seconds`, but UI shows `1 minute, 15 seconds`

![image](https://cloud.githubusercontent.com/assets/119096/25209609/950c494c-25bf-11e7-8bd0-d9d96dc83f03.png)

Task log here: https://deploy3.octopushq.com/app#/tasks/ServerTasks-98947/raw

**Comment by MarcoJ-CID** [comment]
Hi @all.

we updated today our Octopus Server from version 3.7.10 to 3.13.3 and we've got now the same bug.
It's a bit confusing that the UI don't show the correct deployment run time of each single tentacle.
![image](https://cloud.githubusercontent.com/assets/6771839/26249184/c35b6bb4-3ca5-11e7-8787-d6806ec7c038.png)
We've got more than 300 installed tentacles and the update of all take ~10 minutes and not one of them. 
We see that the deployment time for each step is shown correctly but the problem are the child step. a fix of this would be nice.

![image](https://cloud.githubusercontent.com/assets/6771839/26249454/fbfdf170-3ca6-11e7-85fe-e7c001abcf01.png)
